test/unit/api/test_fastapi.py

```diff
@@ -0,0 +1,67 @@
+import uuid
```

Comment:
This file can probably go in `test/unit/webapps` , to be symlinked by the `webapps` package?